examples/pytorch/answer-generator/requirements.txt

```diff
@@ -0,0 +1,3 @@
+torch==1.3.1
```

Comment:
Is it necessary to specify torch version? we decided to not have versions for packages that we provide out of box in predictor container right?